courses/flower_cup/toads_turnpike/course_data.inc.c

```diff
@@ -8386,7 +8386,7 @@ Gfx *d_course_toads_turnpike_dl_list[] = {
 };
 
 // 0x23AE0
-struct ActorSpawnData d_course_toads_turnpike_itembox_spawn[] = {
+struct ActorSpawnData d_course_toads_turnpike_itembox_spawns[] = {
```

Comment:
item_box or itembox. Choose 1.

Reply:
item_box for sure. Must have missed the underscore by accident